static/src/stylesheets/module/content/_interactive.scss

```diff
@@ -62,31 +44,24 @@
                 width: gs-span(6) + ($gs-gutter / 2);
             }
         }
-        .content__standfirst {
-            @include mq(tablet) {
-                float: left;
-                padding-right: $gs-gutter / 2;
-                width: gs-span(7);
-            }
-            @include mq(desktop) {
-                width: gs-span(5) + ($gs-gutter / 2);
-            }
-            @include mq(leftCol) {
-                width: gs-span(6) + ($gs-gutter / 2);
-            }
-            @include mq(wide) {
-                width: gs-span(8) + ($gs-gutter / 2);
-            }
-        }
     }
     .content__meta-container {
         margin-left: 0;
         width: 100%;
+        border-top: 0;
         border-bottom: 0;
 
+        .meta__number {
+            border-left: 1px solid colour(neutral-5);
+            padding-left: $gs-gutter / 2;
```

Comment:
Not vital, but the way this partial switches between `$gs-whatever/2` and `$gs-whatever / 2` is gross.